.github/readmes/typescript/postgis-express/README.md

```diff
@@ -0,0 +1,25 @@
+# Postgres Geolocation example
+
+This example shows how to implement **Geolocation in Postgres** using [Express](https://expressjs.com/), [Postgis](http://postgis.net/) and [Prisma Client](https://github.com/prisma/prisma2/blob/master/docs/prisma-client-js/api.md).
```

Comment:
```suggestion
This example shows how to implement **Geolocation in PostgreSQL** using [Express](https://expressjs.com/), [Postgis](http://postgis.net/) and [Prisma Client](https://github.com/prisma/prisma2/blob/master/docs/prisma-client-js/api.md).
```

Is the "Geolocation" the correct label for this? Or maybe better "Geolocation Types" or similar?